Makefile

```diff
@@ -76,6 +76,10 @@ cobertura:
 
 # Ensure a PR is ready for review.
 reviewable: generate lint
+	@$(INFO) checking go modules are tidy
+	@go mod tidy
+	@git diff --quiet || $(FAIL)
+	@$(OK) go modules are tidy
```

Comment:
Do we want the `make reviewable` command itself to fail if there's a diff (which I presume could be due to `go mod tidy`, `make generate`, or something else entirely)? I frequently run `make reviewable` locally, so I'm not sure how I feel about a diff making it fail. I was thinking more that:

1. We'd invoke `make reviewable` as part of the build.
1. If `make reviewable` resulted in a diff (e.g. because the author forgot to tidy their modules, or generate new code) the build would fail.

Perhaps we'd need to wrap `make reviewable` in a `make check-reviewable` (with a better name?) so something like:

```make
reviewable: generate lint
	@go mod tidy

check-reviewable: reviewable
	@$(INFO) Confirming make reviewable does not produce a git diff
	@git diff --quiet || $(FAIL)
	@$(OK) Confirmed make reviewable does not produce a git diff
```

Reply:
Great point! Thanks for the feedback :) I have updated to accomplish the goals above, let me know what you think!